tests/load/bigquery/test_bigquery_client.py

```diff
@@ -86,6 +86,25 @@ def test_bigquery_job_errors(client: BigQueryClient, file_storage: FileStorage)
     assert r_job.status() == "completed"
 
 
+@pytest.mark.skip()
+def test_bigquery_location(client: BigQueryClient, file_storage: FileStorage) -> None:
+    user_table_name = prepare_event_user_table(client)
+    load_json = {
+        "_dlt_id": uniq_id(),
+        "_dlt_root_id": uniq_id(),
+        "sender_id":'90238094809sajlkjxoiewjhduuiuehd',
+        "timestamp": str(pendulum.now())
+    }
+    #job = expect_load_file(client, file_storage, json.dumps(load_json), user_table_name)
```

Comment:
what is the problem with the test? let's set up the testing so you can run tests locally